projects/fusion-e2e/entities/components/inputs/base-input.ts

```diff
@@ -0,0 +1,57 @@
+import {getTestId} from '../../global/utils';
+import {InputTestIdModifiers} from '@ironsource/fusion-ui/entities/test-ids-modifiers';
+import {BaseComponent} from '../base-component';
+import {Locator, Page} from '@playwright/test';
+
+export class BaseInputComponent extends BaseComponent {
+    constructor(page: Page, selector: string) {
+        super(page, selector);
+    }
+
+    // Get the text of the input field
+    async getInputsFieldText(): Promise<string> {
```

Comment:
you shouldn’t have to use getTest id in your methods - you need to declare class properties and create an instance of them in the cosntruuctor